metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -206,6 +208,8 @@ case class SbtBuildTool(
             shutdownBspServer(shellRunner).ignoreValue
           case _ => Future.successful(())
         }
+        .withTimeout(10, TimeUnit.SECONDS)
```

Comment:
IS this the timeout of message requests in VS Code?

Reply:
More like 15sec. But I also realised that this wasn't very smart, since if you'd press the `restart` button after the timeout (which you always can do) it would simply shut down the sbt server.  I fixed that now.